### PR TITLE
drivers:adc:ad713x:Added a delay after GPIO operations

### PR DESCRIPTION
**Pull request by jansunil**
Added a delay after the GPIO operations as the device needs a 10 ms delay to come out of power down state. The power down operation has been added as a part of the ad713x_init_gpio().

### CODE REVIEW

File: drivers/adc/ad713x/ad713x.c [hunk 1]
```diff
@@ -549,6 +549,8 @@ int32_t ad713x_init(struct ad713x_dev **device,
 	if(NO_OS_IS_ERR_VALUE(ret))
 		goto error_gpio;
 
+	no_os_mdelay(10);
```
Comment (amiclaus) [inline drivers/adc/ad713x/ad713x.c:552]: just curious, is this delay related to the datasheet specifications? 

Also i'd move it in the gpio_init function.

Comment (jansunil) [reply drivers/adc/ad713x/ad713x.c:552]: Yes, this delay has been mentioned in the datashseet for the PDN functionality.

**Comment by jansunil** [comment]
V1: Addressed review comments

Moved the delay to the ad713x_init_gpio() function

**Review by amiclaus — APPROVED** [review]
